src/components/checkbox/checkbox.scss

```diff
@@ -270,7 +270,7 @@ md-checkbox {
   opacity: 0;
   display: inline-flex;
   justify-content: center;
-  transition: background-color $md-checkbox-transition-duration
+  transition-timing-function: background-color $md-checkbox-transition-duration
```

Comment:
I don't think this is the right syntax for `transition-timing-syntax` (including the `background-color` in the value)